rllib/evaluation/collectors/agent_collector.py

```diff
@@ -189,6 +189,12 @@ def add_action_reward_next_obs(self, input_values: Dict[str, TensorType]) -> Non
         values[SampleBatch.OBS] = values[SampleBatch.NEXT_OBS]
         del values[SampleBatch.NEXT_OBS]
 
+        # Default to next timestep but also accept it in input_values
+        if SampleBatch.T not in input_values:
+            self.buffers[SampleBatch.T][0].append(
+                self.buffers[SampleBatch.T][0][-1] + 1
+            )
```

Comment:
it may read slightly better if we just update the input values, like existing surrounding logics, instead of modifying self.buffers directly.
```
if SampleBatch.T not in input_values:
    assert len(self.buffers[SampleBatch.T][0]) > 0
    values[SampleBatch.T] = self.buffers[SampleBatch.T][0][-1]
```
so self.buffers is not updated at multiple scattering places.

Reply:
Done!